Packs/PAN-OS/ReleaseNotes/1_16_23.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Palo Alto Networks PAN-OS
+
+- Fixed documentation link
```

Comment:
```suggestion
- Updated the Docker image to: *demisto/pan-os-python:1.0.0.57650*.
```

do we actually need on the RN to let customers know about changes in `README.md`?